routers/web/org/teams.go

```diff
@@ -126,10 +131,29 @@ func TeamsAction(ctx *context.Context) {
 		u, err = user_model.GetUserByName(ctx, uname)
 		if err != nil {
 			if user_model.IsErrUserNotExist(err) {
-				ctx.Flash.Error(ctx.Tr("form.user_not_exist"))
+				if setting.MailService != nil && user_model.ValidateEmail(uname) == nil {
+					invite, err := org_model.CreateTeamInvite(ctx, ctx.Doer, ctx.Org.Team, uname)
```

Comment:
Huh?
You invite a user that doesn't even exist?
How would that make sense?
I'd understand it on an `ErrUserNotTeamMember`, but on an `ErrNotExist`?

Reply:
That's the whole point of this PR. Inviting users that do not exist yet.